revenuecat_examples/MagicWeather/android/build.gradle

```diff
@@ -1,12 +1,13 @@
 buildscript {
-    ext.kotlin_version = '1.6.21'
+    ext.kotlin_version = '1.7.21'
     repositories {
         google()
         mavenCentral()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:8.0.0'
+        // TODO: Upgrade to gradle 8 when modal_progress_hud_nsn is updated to support it
+        classpath 'com.android.tools.build:gradle:7.4.2'
```

Comment:
Weird, because I am pretty sure I ran this example and it ran fine. What was the issue?

Reply:
Getting this error, which I think it's not really possible to workaround from the host app?

```
A problem occurred configuring project ':modal_progress_hud_nsn'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Please specify a namespace in the module's build.gradle file like so:

     android {
         namespace 'com.example.namespace'
     }
```